components/table/stories/template.js

```diff
@@ -204,65 +285,75 @@ export const Template = ({
 		id=${ifDefined(id)}
 		role=${ifDefined(useCheckboxCell ? "grid" : useDivs ? "table" : undefined)}
 		aria-multiselectable=${ifDefined(useCheckboxCell ? "true" : undefined)}
-		style="max-width: 800px;"
```

Comment:
Should I keep this? Is it necessary? I'm fine with adding this max-width back on, just let me know!